README.md

```diff
@@ -3,6 +3,47 @@ A repository for developing the User Interface for receiving data and controllin
 
 
 ## Packages Info
+# Autolaunch
+
+This guide will show you how to set up and launch the MRCS (Manual Rover Control System) using ROS 2 on a Docker bash terminal.
+
+## Instructions
+
+1. Open your ROS 2 Docker terminal.
+2. Install `tmux` by running `sudo apt-get install tmux`.
+   - `tmux` is a terminal multiplexer that allows you to create multiple windows and panes within a single terminal window.
+3. Create a new directory named `ros2_ws` with a subdirectory called `src` using `mkdir -p ros2_ws/src`.
+4. Navigate to the subdirectory inside the folder you just created by running `cd ros2_ws/src`.
+5. Clone the latest version of the MRCS repo using `git clone git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`.
+6. Change back into `ros2_ws` by running the command `cd ros2_ws`.
+7. Build the project by running `colcon build`.
+   - If you encounter the error message `stderr: teleop_twist_joy`, run `colcon build` again to resolve it.
+8. Source ROS2 in the parent directory by running the command: `. install/setup.bash`.
+9. Startup `tmux` by running the command `tmux`.
+   - If a green status bar appears at the bottom of your terminal, move on to the next step. Otherwise, try step 9 again, and if that doesn't work, try step 1 to make sure it is installed correctly on your system.
+10. Before moving on, note that all commands in `tmux` are triggered by a prefix key followed by a command key. For our use case, the "prefix key" is the keyboard shortcut `CTRL+B`.
+11. Press `CTRL+B` followed by a ` SHIFT+ ‘` or `"` to open a top and bottom split-screen terminal.
```

Comment:
We don't need to teach the user how to use tmux. Put a link to a guide/keybind list on tmux in the note on tmux ctrl b. Just say to open a split screen terminal with tmux.